strr-web/components/bcros/form-section/property/Details.vue

```diff
@@ -104,6 +159,23 @@ const ownershipType = defineModel<string>('ownershipType')
 const businessLicense = defineModel<string>('businessLicense')
 const parcelIdentifier = defineModel<string>('parcelIdentifier')
 const businessLicenseExpiryDate = defineModel<string>('businessLicenseExpiryDate')
+const rentalUnitSpaceType = defineModel<string>('rentalUnitSpaceType')
+const isUnitOnPrincipalResidenceProperty = defineModel<boolean>('isUnitOnPrincipalResidenceProperty')
+const hostResidence = defineModel<string | null>('hostResidence')
+
+if (!formState.propertyDetails.numberOfRoomsForRent) {
+  formState.propertyDetails.numberOfRoomsForRent = 1
```

Comment:
You can set the `formState.propertyDetails.numberOfRoomsForRent` to have initial value of 1 (in `strr.ts`), so this won't be needed.

Reply:
done!